src/content/nft/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Non-fungible tokens (NFT)
+description: An overview of NFTs on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/eth2/core.png
+summaryPoints:
+  [
+    "A way to turn anything unique into an Ethereum-based asset.",
+    "NFTs are giving more power to content creators than ever before.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+NFTs are currently taking the digital art and collectibles world by storm. Digital artists are seeing their lives change thanks to huge sales to a new crypto-audience. And celebrities are joining in as they spot a new opportunity to connect with fans.
+
+Here's what you need to know about NFTs.
+
+## What's an NFT?
+
+NFTs are tokens that we can use to represent a record of ownership. They let us tokenise art, collectibles, even purely physical items like designer trainers/sneakers. They can only have one owner at a time and they're protected by the Ethereum blockchain – no one can modify the record of ownership or copy/paste a new NFT into existence.
```

Comment:
> They let us tokenise art, collectibles, even purely physical items like designer trainers/sneakers.

This is solid but IMO potentially paints too narrow of a picture - that NFTs are primarily focused around art/design/games. Perhaps we throw in a mention of real estate (title to a house?) or financial securities (a government bond?) to demonstrate how broad these applications could be?